packages/api-sdk/src/medical/models/patientDTO.ts

```diff
@@ -7,7 +7,7 @@ export type PatientDTO = {
   firstName: string;
   lastName: string;
   dob: string;
-  genderAtBirth: "M" | "F";
+  genderAtBirth: "M" | "F" | "O" | "U";
```

Comment:
I know we have an upcoming fix for the mismatching types, but reckoned it wouldn't hurt to expose this properly now as a step towards the fix.